content/06.methods.md

```diff
@@ -113,6 +113,12 @@ Note that GATK v4.0.12.0 was used for all of these steps except VariantFiltratio
 This single-sample workflow is available in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows).
 References can be obtained from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket, and a general description of references can be found at https://s3.amazonaws.com/broad-references/broad-references-readme.html.
 
+##### Pathogenicity calling
```

Comment:
I feel like this heading could be made more specific? Perhaps "Identification of pathogenic germline variants"?

Reply:
actually made this more specific to the hypermutant samples since we did not assess across the entire PBTA here (separate ongoing work)